docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,35 +263,100 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
-. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
+Use {ref}/esql.html[ES|QL] to query your source events and aggregate event data. Query results are returned in a table with rows and columns. Each row becomes an alert.
+
+To create an ES|QL rule:
 
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
+. Select **ES|QL**, then write a <<esql-rule-query-types,query>>. 
 +
-[esql]
------
+TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help icon,20,20]) to open the in-product reference documentation for all ES|QL commands and functions.
++
+. Click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
+
+[float]
+[[esql-rule-query-types]]
+==== ES|QL query types 
+
+ES|QL rule queries are loosely categorized into two types: aggregating and non-aggregating. 
+
+[float]
+[[esql-agg-query]]
+===== Aggregating query 
+
+An aggregating query uses {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Generated alerts only contain the fields returned by the query.
```

Comment:
```suggestion
Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. The alerts they generate only contain the fields returned by the query.
```
If this is accurate I think it's a bit more clear, but I could be wrong!

Reply:
From a technical perspective, it might be more accurate to say that the rule generates alerts -- at least that's what I'm gathering from @vitaliidm's explanation [here](https://github.com/elastic/security-docs/pull/4062#discussion_r1374795992). 

How does this look, @benironside:
```suggestion
Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by the {esql} rule only contain the fields returned by the query.
```

Reply:
```suggestion
Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by an {esql} rule only contain the fields returned by its query.
```

Reply:
What about this?